testutil/network/network.go

```diff
@@ -248,7 +248,9 @@ func New(l Logger, baseDir string, cfg Config) (*Network, error) {
 
 		ctx := server.NewDefaultContext()
 		tmCfg := ctx.Config
-		tmCfg.Consensus.TimeoutCommit = cfg.TimeoutCommit
+		// Reduce the target height duration so that blocks are produced faster
+		// during tests.
+		tmCfg.Consensus.TargetHeightDuration = time.Millisecond * 300
```

Comment:
Is this actually used by us?

Reply:
there are no references in this repo so it's possible it's unneeded. Checking now